### PR TITLE
Unit test for binomial distribution

### PR DESCRIPTION
**Pull request by mahausmani**
This PR is partially addressing #1

### CODE REVIEW

File: tests/drvs_test.py [hunk 1]
```diff
@@ -14,19 +14,70 @@
...
 
-
 sys.path.append("../jaxampler")
+import sys
+import pytest
+import jax
+import jax.numpy as jnp
+from jaxampler._src.rvs.binomial import Binomial
 
 eps = 1e-3
```
Comment (Qazalbash) [inline tests/drvs_test.py:24]: Unused variable. Thanks for pointing out the usage of `jnp.allclose`.

File: tests/drvs_test.py [hunk 2]
```diff
@@ -14,19 +14,70 @@
...
+            Binomial(p=0.1, n=100000).pmf_x(50),
+            jax.scipy.stats.binom.pmf(50, 100000, 0.1),
+        )
 
+    def test_cdf_x(self):
+        binomial = Binomial(p=0.2, n=12)
+        assert binomial.cdf_x(x=13) == 1
+        assert binomial.cdf_x(-1) == 0
+        assert binomial.cdf_x(9) >= 0
+        assert binomial.cdf_x <= 1
```
Comment (Qazalbash) [inline tests/drvs_test.py:70]: There should be an argument to the function.

**Review by Qazalbash — CHANGES_REQUESTED** [review]
PR is missing key things,
- [ ] Meaningful variable names.
- [x] Edge cases.
- [ ] Functions missing arguments.
Fix the unchecked points and resubmit. Thanks!

**Review by Qazalbash — CHANGES_REQUESTED** [review]
Same reviews as before

File: tests/drvs_test.py [hunk 3]
```diff
@@ -16,68 +16,68 @@
 
 sys.path.append("../jaxampler")
 import sys
```
Comment (Qazalbash) [inline tests/drvs_test.py:18]: Sorted imports are good.

File: tests/drvs_test.py [hunk 4]
```diff
@@ -16,68 +16,68 @@
...
-        assert jnp.allclose(Binomial(p=0.5, n=10).logpmf_x(5), jax.scipy.stats.binom.logpmf(5, 10, 0.5))
+        assert jnp.allclose(
+            Binomial(p=0.5, n=10, name="test_logpmf_p0.5").logpmf_x(5), jax.scipy.stats.binom.logpmf(5, 10, 0.5))
 
-        assert Binomial(p=0.5, n=[10, 20]).logpmf_x(5).shape == (2,)
-        assert Binomial(p=[0.5, 0.1], n=[10, 20]).logpmf_x(5).shape == (2,)
-        assert Binomial(p=[0.5, 0.1, 0.3], n=[10, 20, 30]).logpmf_x(5).shape == (3,)
+        assert Binomial(p=0.5, n=[10, 20], name="test_logpmf_n2").logpmf_x(5).shape == (2,)
+        assert Binomial(p=[0.5, 0.1], n=[10, 20], name="test_logpmf_p2n2").logpmf_x(5).shape == (2,)
+        assert Binomial(p=[0.5, 0.1, 0.3], n=[10, 20, 30], name="test_logpmf_p3n3").logpmf_x(5).shape == (3,)
```
Comment (Qazalbash) [inline tests/drvs_test.py:34]: Naming variables is also a good check.

**Review by Qazalbash — APPROVED** [review]
Requested changes completed. Thanks for the PR.

**Comment by Qazalbash** [comment]
@mahausmani test has failed, but they are due to the internal framework. Thank you for this contribution and for being the first community member to recognize this project.

**Comment by Qazalbash** [comment]
@mahausmani we are only obliged to test those functions, which are at their core written by us and serve our purpose for example in the case of `rvs.Binomial`, `logpmf_x` is written as a wrapper over `jax.scipy.stats.binom.logpmf`, but `logcdf_x` is a result of our intuition and logic. Therefore we will only test the `logcdf_x` not `logpmf_x`.
